### PR TITLE
Create to do list structure

### PR DESCRIPTION
**Pull request by JOdiemo**
Use webpack to bundle JavaScript.
Learn how to use proper ES6 syntax.
Use ES6 modules to write modular JavaScript.

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that 

- [ ] has README file that is not customized to your project. Kindly customise the readme file according to your project requirement. All the headers must be present in your customized one with their appropriate information under them except FAQ which is optional. Your must remove the Microverse logo and replace it with your logo if you have any.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -241,48 +86,22 @@ Feel free to check the [issues page](../../issues/).
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./license) licensed.
```
Comment (GzimAsani) [inline README.md]: - [ ] The **MIT**  link leads to a 404 page, kindly update it.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+    "hint": "^7.1.6",
+    "html-webpack-plugin": "^5.5.0",
+    "lodash": "^4.17.21",
+    "style-loader": "^3.3.2",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-server": "^4.13.1"
```
Comment (GzimAsani) [inline package.json:619]: - [ ] You did not install webpack as devDependency, and you need that in order to run your project with npm start. Kindly install it with `npm install webpack --save-dev.`
image

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <div class="heading">
+        <h4 class="title">Today's To do</h4>
+        <i class="fa fa-refresh fresher" aria-hidden="true"></i>
+        </div>
+        <input type="text" placeholder="Add List Items."/>
+        <div class="todos">
+            <!-- Add list dynamicaly /src/index.js -->
+        </div>
+        <div class="clear">
+            <a>Clear All Completed</a>
```
Comment (GzimAsani) [inline src/index.html:24]: - [ ] Kindly style the button similar to the given video design 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @JOdiemo            :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the iteration and object creation :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @JOdiemo 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Professional readme file
✔️ Clean Codebase
✔️ No linters error

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**